<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>641</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="mainLayout">
     <item>
      <widget class="StackList" name="stakcList" native="true">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="pages">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="pageWelcome">
        <layout class="QVBoxLayout" name="welcomeLayout">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>12</number>
         </property>
         <item>
          <widget class="QLabel" name="label1">
           <property name="font">
            <font>
             <family>Sans Serif</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>IP Addresses Available</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layout1">
           <item>
            <widget class="QTextBrowser" name="textBrowser">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="vlSetting1">
             <item>
              <widget class="QPushButton" name="btnAddrSettings">
               <property name="text">
                <string>Settings...</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label2">
           <property name="font">
            <font>
             <family>Sans Serif</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Emulation State</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layout2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="lblTxRate">
               <property name="text">
                <string>TxRate: 100Mbps</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblRxRate">
               <property name="text">
                <string>RxRate: 100Mbps</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="vlSettings">
             <item>
              <widget class="QPushButton" name="btnEmuSettings">
               <property name="text">
                <string>Settings...</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label3">
           <property name="font">
            <font>
             <family>Sans Serif</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Statistics</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="layout3">
           <item>
            <widget class="QLabel" name="lblScenarios">
             <property name="text">
              <string>Scenarios: 0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblScripts">
             <property name="text">
              <string>Scripts: 0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblTests">
             <property name="text">
              <string>Tests: 0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageScenario"/>
       <widget class="QWidget" name="pageScript">
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>261</width>
           <height>27</height>
          </rect>
         </property>
        </widget>
        <widget class="QTextEdit" name="textEdit">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>261</width>
           <height>221</height>
          </rect>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>10</y>
           <width>108</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Setup Script</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>10</y>
           <width>108</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Reset Script</string>
         </property>
        </widget>
        <widget class="QListView" name="listView">
         <property name="geometry">
          <rect>
           <x>285</x>
           <y>50</y>
           <width>231</width>
           <height>221</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pageNewTest"/>
       <widget class="QWidget" name="pageViewTests"/>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionWelcome"/>
    <addaction name="actionScenario"/>
    <addaction name="actionScript"/>
    <addaction name="actionNewTest"/>
    <addaction name="actionViewTests"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHomepage"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHomepage">
   <property name="text">
    <string>Go to Homepage</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionWelcome">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Welcome</string>
   </property>
  </action>
  <action name="actionScenario">
   <property name="text">
    <string>Scenario</string>
   </property>
  </action>
  <action name="actionScript">
   <property name="text">
    <string>Script</string>
   </property>
  </action>
  <action name="actionNewTest">
   <property name="text">
    <string>New Test</string>
   </property>
  </action>
  <action name="actionViewTests">
   <property name="text">
    <string>View Tests</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StackList</class>
   <extends>QWidget</extends>
   <header>stacklist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
